lib/pages/home/home.dart

```diff
@@ -40,7 +40,7 @@ class HomePage extends StatelessWidget {
               Icons.view_headline,
               key: ValueKey(SideDrawerKeys.DRAWER),
             ),
-            onPressed: () => _scaffoldKey.currentState.openDrawer()),
+            onPressed: () => _scaffoldKey.currentState!.openDrawer()),
```

Comment:
Why we are using `!.openDrawer()` operator instead of `?.openDrawer()` here ?

Reply:
Makes sense, thanks for pointing out.